python/paddle/fluid/tests/unittests/dygraph_to_static/test_mnist.py

```diff
@@ -153,6 +153,18 @@ def test_mnist_declarative(self):
             msg='dygraph is {}\n static_res is \n{}'.format(dygraph_loss,
                                                             static_loss))
 
+    def test_mnist_declarative_cpu_vs_mkldnn(self):
+        dygraph_loss_cpu = self.train_dygraph()
+        fluid.set_flags({'FLAGS_use_mkldnn': True})
```

Comment:
Setting the global flag here can affect other tests when they are run in parallel, can't it?

Reply:
I overlooked this. I think it can affect other tests because gflags are static and ctest runs parallel tests within single process.
However, other tests do change globals too:
https://github.com/PaddlePaddle/Paddle/blob/ce506930c39c070105ca7861459764bd6cff6984/python/paddle/fluid/tests/unittests/test_activation_op.py#L1272
https://github.com/PaddlePaddle/Paddle/blob/ce506930c39c070105ca7861459764bd6cff6984/python/paddle/fluid/tests/unittests/test_auto_growth_gpu_memory_limit.py#L22
https://github.com/PaddlePaddle/Paddle/blob/ce506930c39c070105ca7861459764bd6cff6984/python/paddle/fluid/tests/unittests/test_eager_deletion_delete_vars.py#L17

Reply:
OK, I think the time spent with the flag set is minimized.